core/parameters/res/runtime_configs/69.yaml

```diff
@@ -1,3 +1,22 @@
+# State Witness size limits.
+
+max_transaction_size: {old: 4_194_304, new: 1_572_864}
+
+per_receipt_storage_proof_size_limit: {old: 999_999_999_999_999, new: 4_000_000}
+main_storage_proof_size_soft_limit: {old: 999_999_999_999_999, new: 3_000_000}
+
+max_receipt_size: {old: 999_999_999_999_999, new: 4_194_304}
+new_transactions_validation_state_size_soft_limit: {old: 999_999_999_999_999, new: 572_864}
+
+# 100 kiB
+outgoing_receipts_usual_size_limit: {old: 999_999_999_999_999, new: 102_400}
+
+# 4.5 MiB
+outgoing_receipts_big_size_limit: {old: 999_999_999_999_999, new: 4_718_592}
+
+combined_transactions_size_limit: {old: 999_999_999_999_999, new: 4_194_304}
+
+
 # Change the cost of sending receipt to another account to 50 TGas / MiB
```

Comment:
where does this come from? I would expect 68.yaml be an as-is copy of 80.yaml (which is already the case) and 69.yaml be an as-is copy of 81.yaml.

Reply:
This is from the new limits introduced in stateless validation

Reply:
@tayfunelmas This file is a merge of 81.yaml and 82.yaml